gdx/src/com/badlogic/gdx/math/Polygon.java

```diff
@@ -175,6 +174,23 @@ public float area () {
 		return GeometryUtils.polygonArea(vertices, 0, vertices.length);
 	}
 
+	 public int getNbVertices(){

```

Comment:
This isn't a good name. Do you mean `getNumberVertices`? I would use `getVertexCount`.

Reply:
yes, thanks
done